test/UnitTest/data/cosmoscollections/dataelements/11f7c994-6681-47a1-9626-fcf6c27308a5.json

```diff
@@ -9,5 +9,7 @@
   "refs": [
   ],
   "created": "2020-05-11T17:09:28.4621953Z",
-  "lastChanged": "2020-05-11T17:09:28.4621953Z"
+  "lastChanged": "2020-05-11T17:09:28.4621953Z",
```

Comment:
Is it still relevant to update a cosmoscollection of testdata? Do we need to rename the folder?